packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts

```diff
@@ -38,30 +39,23 @@ export class CreateOrUpdateBotFrameworkBotDriver implements StepDriver {
     try {
       this.validateArgs(args);
 
-      const tokenResult = await context.m365TokenProvider.getAccessToken({
-        scopes: AppStudioScopes,
-      });
-      if (tokenResult.isErr()) {
-        throw tokenResult.error;
-      }
+      const botRegistrationData: IBotRegistration = {
+        botId: args.botId,
+        name: args.name,
+        description: args.description ?? "",
+        iconUrl: args.iconUrl ?? "",
+        messagingEndpoint: args.messagingEndpoint,
+        callingEndpoint: "",
+      };
+
+      const botRegistration: BotRegistration = new LocalBotRegistration();
```

Comment:
Is it possible to make `LocalBotResitration` a class property or static variable?
Creating new instance every time seems not a good practice.

Reply:
I agree. There's a `factory` which may return a static instance everytime, I may use it in another PR.